docs/source/rule-testing.rst

```diff
@@ -6,36 +6,41 @@ To test the accuracy of new rules, local tests can be written to verify that ale
 Configuration
 ~~~~~~~~~~~~~
 
-To test a new rule, first create a new file under ``tests/integration/rules`` named ``rule_name_goes_here.json``.  This file should contain this exact structure::
+To test a new rule, first create a new JSON file anywhere within ``tests/integration/rules`` named ``test_records.json``.  This file should contain this exact structure::
```

Comment:
The old name (`rule_name_goes_here `) denoted they should pick the name, whereas your new text makes that less apparent

Reply:
I'll add a comment about picking a name. I didn't want to leave it as `rule_name_goes_here` because the requirement for them being named after the rule no longer exists